pkg/bbgo/strategy_controller.go

```diff
@@ -0,0 +1,119 @@
+package bbgo
+
+import (
+	"fmt"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	"reflect"
+)
+
+type StrategyController struct {
+	Status types.StrategyStatus
+
+	// Callbacks
+	GetStatusCallback     func() types.StrategyStatus
```

Comment:
I think we don't need this GetStatusCallback~ because we don't need to override it or register a callback on it.

Reply:
done